src/backend/TrafficCourts/Citizen.Service/Validators/DisputedCountValidator.cs

```diff
@@ -27,17 +27,17 @@ public DisputedCountValidator()
                 .WithMessage("'Appear In Court' selection is required to be set since disputant pleaded guilty");
 
             // Validation rules for properties if disputant pleaded not guilty
-            RuleFor(_ => _.RequestTimeToPay).Null()
+            RuleFor(_ => _.RequestTimeToPay).Must(x => x.Value == false || x is null)
                 .When(_ => _.Plea.Equals(Plea.NotGuilty))
-                .WithMessage("'Request Time To Pay' must not be set since disputant pleaded not guilty");
+                .WithMessage("'Request Time To Pay' must not be true since disputant pleaded not guilty");
 
-            RuleFor(_ => _.RequestReduction).Null()
+            RuleFor(_ => _.RequestReduction).Must(x => x.Value == false || x is null)
                 .When(_ => _.Plea.Equals(Plea.NotGuilty))
-                .WithMessage("'Request Reduction' must not be set since disputant pleaded not guilty");
+                .WithMessage("'Request Reduction' must not be true since disputant pleaded not guilty");
 
-            RuleFor(_ => _.AppearInCourt).Null()
+            RuleFor(_ => _.AppearInCourt).Must(x => x.Value == true || x is null)
                 .When(_ => _.Plea.Equals(Plea.NotGuilty))
-                .WithMessage("'Appear In Court' must not be set since disputant pleaded not guilty");
+                .WithMessage("'Appear In Court' must not be false since disputant pleaded not guilty");
```

Comment:
At some point, it would be nice to have Xunit tests to confirm the logic here for these validator rules.